src/analytics/private/metrics-common-workflow.ts

```diff
@@ -123,6 +136,7 @@ export function buildMetricsWidgetForWorkflows(scope: Construct, id: string, pro
     [loadDataWorkflowDimension, 'Load data to redshift tables'],
     [clearExpiredEventsWorkflowDimension, 'Clear expired events'],
     [upsertUsersWorkflowDimension, 'Upsert user'],
+    [scanMetadataWorkflowDimension, 'scan metadata'],
```

Comment:
update to `Scan metadata` consistent with other metrics

Reply:
Done